pkg/services/vmoperator/vmoperator_suite_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package vmoperator
+package vmoperator_test
```

Comment:
q: why are moving this to a _test, AFAIK this is not necessary

Reply:
answered above, reverted.